test/events/services/service_deleted_event_test.rb

```diff
@@ -18,15 +18,17 @@ def test_ability
     member  = FactoryGirl.build_stubbed(:simple_user, account: service.account,
                                         admin_sections: [:partners, :services])
 
+    member.member_permission_service_ids = [service.id]
```

Comment:
This is related to filtering only the "existing" services on saving. This is a change introduced in the PR that resulted in breaking this test.

The new behavior is desired, however, so the test is fixed by changing the order – first setting the allowed service IDs, then deleting the service.